tests/ChunkingRedisClient.Tests/TestObjects/TestDataGenerator.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
+
+namespace ChunkingRedisClient.Tests.TestObjects
+{
+    public class TestDataGenerator
```

Comment:
AutoFixture?

Reply:
I'd recommend keeping it simple, if the test object becomes anymore complex AND as long as we don't assert on the values generated by AutoFixture in our tests (which can make tests flaky), then sure using AutoFixture is a fine thing to do. Adding more packages just adds more and more entropy that we have little control over and considering the simple object structure, I'd say for now using AF is an over-kill. Does that make sense?